broker/startup_checks_test.go

```diff
@@ -64,6 +65,23 @@ var _ = Describe("Initializing the broker", func() {
 			Expect(brokerCreationErr).To(HaveOccurred())
 			Expect(brokerCreationErr.Error()).To(ContainSubstring("Cloud Foundry API version couldn't be parsed. Expected a semver, got: 1.invalid.0"))
 		})
+
+		Context("when CF startup checks are disabled", func() {
+			It("does not verify CF API version", func() {
+				_, brokerCreationErr := broker.New(
+					boshInfo,
+					boshClient,
+					cfClient,
+					serviceAdapter,
+					fakeDeployer,
+					serviceCatalog,
+					true,
+					loggerFactory,
+				)
+				Expect(brokerCreationErr).NotTo(HaveOccurred())
+				Expect(cfClient.GetAPIVersionCallCount()).To(Equal(0))
+			})
+		})
```

Comment:
I know that ODB tests structure is not consistent across the project. However, I think it would be better to stick with the style of each file. For instance in this case we could avoid the nested `Context` and add an `It` directly. What do you think?